pycbc/waveform/nr_waveform.py

```diff
@@ -43,12 +43,100 @@ def get_data_from_h5_file(filepointer, time_series, key_name):
     knots = filepointer[key_name]['knots'][:]
     data = filepointer[key_name]['data'][:]
     # Check time_series is valid
-    assert(knots[0] <= time_series[0])
-    assert(knots[-1] >= time_series[-1])
```

Comment:
This catches an existing float [over,under]flow error where knots[0] becomes float precision less than time_series[0]